jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolverTest.kt

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.dotnet
+
+import com.jetbrains.rider.test.asserts.shouldBe
+import com.jetbrains.rider.test.base.BaseTestWithSolution
+import com.jetbrains.rider.test.framework.frameworkLogger
+import org.testng.annotations.DataProvider
+import org.testng.annotations.Test
```

Comment:
That's correct. I'm afraid we should have TestNG dependency since Rider Test Framework is written using TestNg. Tests for Rider a bit differ from Idea tests because we have IDEA and ReSharper together that split some responsibilities. To correctly run tests we should have both up and running since some logic runs in ReSharper process. Idea test framework does not cover this. So, we need to use our Rider Test Framework for testing.

Reply:
Makes sense, wish the fixtures in IntelliJ testFramework was more test framework agnostic, /shrug 😄